packages/templating/plugin/html_scanner.js

```diff
@@ -169,8 +171,15 @@ html_scanner = {
           templateDotNameLiteral + ", " + renderFuncCode + ");\n";
       } else {
         // <body>
-        if (hasAttribs)
-          throwParseError("Attributes on <body> not supported");
+        if (hasAttribs) {
+          for(var attr in attribs) {
+            if (this.bodyAttributes.indexOf(attr) !== -1) {
+              console.log('WARNING: tag ' + attr + ' already defined on body.');
```

Comment:
What happens if you define multiple of the same attribute? I guess the later ones would override the previous ones? I feel like since there is no good way to control the attribute order this should be an error and not a warning unless we can guarantee the same reasonable thing happens every time.